ansible/roles/jupyterhub/files/jupyterhub_config.py

```diff
@@ -137,16 +137,16 @@
 
 # Our user list for demos when using FirstUseAuthenticator. Uncomment and add initial
 # users as needed. This avoids having the login form which accepts any username/password
-c.Authenticator.whitelist = [
-    'admin',
-    'instructor1',
-    'instructor2',
-    'student1',
-    'bitdiddle',
-    'hacker',
-    'reasoner',
-    os.environ.get('DEMO_GRADER_NAME'),
-]
+# c.Authenticator.whitelist = [
```

Comment:
Since we should leave the firstuseauthenticator enabled by default, let's re-enable this whitelist.